presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/type/HiveFileTimestamp.java

```diff
@@ -16,14 +16,14 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
-public class DecodedTimestamp
+public class HiveFileTimestamp
```

Comment:
Either move this into the Hive connector or revert this change -- we shouldn't have a Hive specific class name in the `plugin-toolkit` module.

Reply:
This type of rename should be done as a separate commit, as that makes it easier to review.